Tests/MongoSwiftTests/AsyncAwaitTests.swift

```diff
@@ -58,6 +58,32 @@ extension MongoSwiftTestCase {
         defer { client.syncCloseOrFail() }
         return try await f(client)
     }
+
+    // swiftlint:disable large_tuple
+    // see: https://github.com/realm/SwiftLint/issues/3753
+    internal func withTestNamespace<T>(
+        ns: MongoNamespace? = nil,
+        collectionOptions: CreateCollectionOptions? = nil,
+        _ f: (MongoClient, MongoDatabase, MongoCollection<BSONDocument>) async throws -> T
```

Comment:
this is the line causing the problem with the large_tuple rule; I realized this must be a bug because SwiftLint did not complain about the non-async/await version of this helper which has the same three-parameter method (but non-`async`) as a parameter.